standard/patterns.md

```diff
@@ -158,7 +161,7 @@ The following rules define when a set of patterns is *exhaustive* for a type:
 
 A set of patterns `Q` is *exhaustive* for a type `T` if any of the following conditions hold:
 
-1. `T` is an integral or enum type, or a nullable version of one of those, and for every possible value of `T`’s underlying type, some pattern in `Q` would match that value; or
+1. `T` is an integral or enum type, or a nullable version of one of those, and for every possible value of `T`’s nullable underlying type, some pattern in `Q` would match that value; or
```

Comment:
"Nullable underlying type" sounds like a contradiction in terms to me. (The underlying type isn't nullable, T is.) Would "non-nullable underlying type (if `T` is a nullable type)" be better? Happy to keep this as it is if I'm missing something. (Currently thinking through a Covid brain fog, so may well not be clear...)

Reply:
It was just my typo. Updated.